mocha-client-tests/addon/tests/lib/background-messaging.test.js

```diff
@@ -0,0 +1,10 @@
+describe('Background', function() {
+    describe('ping', function() {
+        it('should return pong in response', function(done) {
+            chrome.runtime.sendMessage({action: 'ping'},function(response) {
+                expect(response).to.equal('pong');
```

Comment:
ok, I see what the issue is now. The `expect()` call throws an error but that is not passed to `done(error)` so the mocha reporter can't show it. The way to fix this is to rewrite the code using promises:
````javascript
it('should return pong in response', function() {
  // Return a promise for Mocha using the Firefox browser API instead of chrome.
  return browser.runtime.sendMessage({action: 'ping'})
    .then(function(response) {
      expect(response).to.equal('pong');
    });
});
````
I think it would be good to convert this code into promises since the developer can always use a [polyfill](https://github.com/mozilla/webextension-polyfill) to make it compatible with Chrome.